CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
```

Comment:
CFF has different requirements which information are necessary depending on the applied version of the standard.  This is the latest version.  The information about the applied CFF version mandatory.